src/install/lockfile.zig

```diff
@@ -1166,10 +1184,10 @@ pub const Printer = struct {
             try writer.writeAll(
                 \\# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
                 \\# yarn lockfile v1
-                \\# bun ./bun.lockb --hash: 
+                \\# bun ./bun.lockb --hash:
             );
             try writer.print(
-                "{}\n\n",
+                " {}\n\n",
```

Comment:
Sorry about this unrelated change &minus; my editor has been configured to trim trailing whitespace on save, and on second thought rather than restoring it I thought this makes said whitespace more readable :thinking: